test/unit_tests/test_scoring.py

```diff
@@ -220,3 +237,45 @@ def on_epoch_end(self, net, *args, **kwargs):
     )
 
     clf.fit(train_set, y=None, epochs=4)
+
+
+def check_preds_supercrops_trials(preds, supercrop_inds, expected_trial_preds):
```

Comment:
```suggestion
def _check_preds_supercrops_trials(preds, supercrop_inds, expected_trial_preds):
```

it makes it clear that this function is a private/aux function from otherones. NOt to be called directly

Reply:
Thanks, followed your suggestion in latest commit.